packages/kuma-gui/src/app/data-planes/routes.ts

```diff
@@ -67,7 +67,29 @@ export const routes = () => {
             {
               path: ':dataPlane',
               name: 'data-plane-summary-view',
+              props: () => ({
+                routeName: 'data-plane-summary-view',
+              }),
+              redirect: { name: 'data-plane-summary-overview-view' },
               component: () => import('@/app/data-planes/views/DataPlaneSummaryView.vue'),
+              children: [
+                {
+                  path: 'summary-overview',
```

Comment:
Note: I went for `/summary-overview` route name as `/overview` is already taken by the `DataPlaneDetailView`

Reply:
Generally it worked for me having both using `/overview`, but when it comes to deeplinking it always went for the closest route, which is the summary. So in order to allow deeplinking to the details overview I had to use different path names 🤔 I might have missed something, so I am looking forward to discuss this further 🙂